src/.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: julia
+
+julia:
+  - 0.4
+  - 0.5
+  - nightly
+
+matrix:
+  fast_finish: true
+  allow_failures:
+    - julia: nightly
+
+notifications:
+  email: false
+
+script:
+  - julia -e 'Pkg.clone(pwd())'
+  - julia --check-bounds=yes --inline=no -e 'Pkg.test("{[{package}]}",
+    coverage=true)'
+
+after_success:
+  - julia -e 'cd(Pkg.dir("{[{package}]}")); Pkg.add("Coverage"); using Coverage;
```

Comment:
`{[{package}]}` place holder.

Well, if you enable CI through Travis for this repo, you will notice this yourself. Maybe it's better to just enable it and see how the build goes.